patch/gfx9/llpcNggPrimShader.cpp

```diff
@@ -6070,10 +6070,10 @@ Function* NggPrimShader::CreateSphereCuller(
         auto pTT = m_pBuilder->CreateInsertElement(pST, pT, static_cast<uint64_t>(0));
 
         // s * <x10, y10> + <x0", y0">
-        auto pXY = m_pBuilder->CreateIntrinsic(Intrinsic::fma, m_pContext->Float16x2Ty(), { pSS, pX10Y10, pX0Y0 });
+        auto pXY = m_pBuilder->CreateIntrinsic(Intrinsic::fma, VectorType::get(Type::getHalfTy(*m_pContext), 2), { pSS, pX10Y10, pX0Y0 });
```

Comment:
This line might exceed 120-character. Please review this commit since the type getter might introduce more characters and increase the line length.